PublicGitArchive/pga/cmd/cache.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-const indexURL = "http://pga.sourced.tech/csv"
+const indexURL = "http://pga.sourced.tech/csv/"
```

Comment:
We shall always use `path.Join` to concat URLs (note: it is different from `filepath`)

Reply:
I did it at first but is snips on of the `/` in the url, returning `http:/pga.sourced.tech/csv/uast`

Reply:
oke doing like https://stackoverflow.com/questions/34668012/combine-url-paths-with-path-join